setup.py

```diff
@@ -12,18 +13,21 @@
 http://www.ucs.louisiana.edu/~kxk4695/statcalc/pois2pval.for
 """
 
-fortran_ext = Extension(
-    "poisson_etest.poisson_etest_fortran", sources=["lib/poisson_etest.f"]
-)
+if __name__ == "__main__":
+    from numpy.distutils.core import setup, Extension
```

Comment:
why a numpy specific distutils/setuptools implementation here?

Reply:
Its to allow the fortran compiler to work I think - This I took from the original repo. 

I also tried with regular `setuptools.setup` but it hit a fortran error